### PR TITLE
MAINT: update to 2017.12

### PR DESCRIPTION
**Pull request by thermokarst**

### CODE REVIEW

**Comment by ebolyen** [comment]
This is going to wait on some of the conda 4.4 business to sort itself out.

File: roles/qiime2/tasks/main.yml [hunk 1]
```diff
@@ -19,28 +19,45 @@
...
+  become: true
+  get_url:
+    url: https://data.qiime2.org/distro/core/qiime2-{{ qiime2_release }}-py35-linux-conda.yml
+    dest: /etc/env.yml
+  when: not env.stat.exists
+
+- name: install qiime2
+  environment:
+    PATH: "{{ miniconda_path }}/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin"
+    TMPDIR: "{{ miniconda_path }}/../tmp"
```
Comment (ebolyen) [inline roles/qiime2/tasks/main.yml]: Why do we need a different /tmp for this operation?

Comment (thermokarst) [reply roles/qiime2/tasks/main.yml]: I ran into FS permission related errors while debugging the memory-related errors. It is possible that this change is ultimately unnecessary, but, it does work as-is right now...

Comment (thermokarst) [reply roles/qiime2/tasks/main.yml]: ☠️ BRB

**Comment by thermokarst** [comment]
Thanks @ebolyen, back in your court 🏀

**Pull request merged**